### PR TITLE
Adding insurance prices to booking

### PR DESCRIPTION
**Pull request by RobinCsl**
@tbergq Do you think I should add an entry to the response corresponding to the 'none' or 'no insurance' case (in which case it is free), instead of handling the special case in the app itself?

### CODE REVIEW

File: src/booking/Booking.js [hunk 1]
```diff
@@ -102,3 +103,8 @@ export type BookingAssets = {
   boardingPasses: BoardingPass[],
   legs: Leg[],
 };
+
+export type InsurancePrice = {|
+  +type: 'travel_basic' | 'travel_plus',
```
Comment (tbergq) [inline src/booking/Booking.js]: There are also deprecated types `skysilver` and `skygold`
https://github.com/kiwicom/graphql/blob/master/src/booking/types/outputs/Passenger.js#L65

Comment (RobinCsl) [reply src/booking/Booking.js]: The way I resolve the insurance prices, I filter on the type and consider only the non-deprecated `travel_basic` and `travel_plus`. Is that a bad thing?

File: src/booking/types/outputs/Passenger.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ import type { Passenger } from '../../Booking';
 import GraphQLVisa from './Visa';
 import GraphQLPkpass from './Pkpass';
 
-const InsuranceTypeEnum = new GraphQLEnumType({
+export const InsuranceTypeEnum = new GraphQLEnumType({
```
Comment (tbergq) [inline src/booking/types/outputs/Passenger.js]: Maybe we should move this to a separate file, now that more types are using it?

File: src/booking/Booking.js (hunk 1 above)
Comment (tbergq) [reply src/booking/Booking.js]: 🤔 It is probably ok. I think you are right.

**Comment by tbergq** [comment]
> Do you think I should add an entry to the response corresponding to the 'none' or 'no insurance' case (in which case it is free), instead of handling the special case in the app itself?

It sounds like a good idea yes.

File: src/booking/types/outputs/Passenger.js (hunk 2 above)
Comment (RobinCsl) [reply src/booking/types/outputs/Passenger.js]: Ok, will do that.

File: src/booking/dataloaders/ApiSanitizer.js [hunk 3]
```diff
@@ -347,3 +352,17 @@ function detectType(apiData): BookingType {
 
   return 'BookingOneWay';
 }
+
+function parseInsurancePrices(insurancePrice, currency): InsurancePrice[] {
+  const types = Object.keys(insurancePrice);
+
+  return types
+    .filter(type => type === 'travel_plus' || type === 'travel_basic')
```
[the remark this answers is not in the document]
Comment (RobinCsl) [reply src/booking/dataloaders/ApiSanitizer.js]: > 🤔 It is probably ok. I think you are right.

I will ignore this comment then 😉

**Comment by RobinCsl** [comment]
@tbergq 🚀

**Pull request merged**